lab-regan/app/directive/social-icons.html

```diff
@@ -0,0 +1,6 @@
+<div class="social-media-icons">
+  <h3>{{socialIconsCtrl.tester}}</h3>
+  <ul>
+    <li><a class="social-icon" ng-repeat="icon in socialIconsCtrl.icons" href="{{icon}}.com"></a></li>
```

Comment:
-This works with the ng-repeat.  Another option would be to dynamically generate a class name in here from what is stored in your icons array.  
-The href attribute will automatically append the attribute to the domain of your page unless you provide it with the new domain.  So if your page is running on http://localhost:8000, and you provide a value of twitter.com in the href attribute, clicking on this element will send you to http://localhost:8000/twitter.com.  Which is not a path that you have setup on your frontend.  To get it to go to https://www.twitter.com you need to provide it with the https://www part.  The https:// will tell it that you are providing an external link.  So you could change the values in your array.  Or if the links you wanted all had the same prefix you could do something like this-->> href='https://www.{{icon}}.com'.